modules/tf_cloudbuild_workspace/variables.tf

```diff
@@ -168,12 +175,12 @@ variable "tf_repo_dir" {
 }
 
 variable "tf_repo_type" {
-  description = "Type of repo"
+  description = "Type of repo. When the repo type is CLOUDBUILD_V2_REPOSITORY, it will use the generig Cloudbuild 2nd gen Repository API."
```

Comment:
```suggestion
  description = "Type of repo. When the repo type is CLOUDBUILD_V2_REPOSITORY, it will use the generic Cloudbuild 2nd gen Repository API."
```

Reply:
fixed